### PR TITLE
Remove getopt, try 2

### PR DESCRIPTION
**Pull request by johnjbarton**
Remove the unused call to getopt and getopt.

### CODE REVIEW

File: build/expand-js-template.js [hunk 1]
```diff
@@ -90,33 +89,6 @@ function expandFile(file, outDir, outStream, options, includeStack) {
 var cmdName = path.basename(process.argv[1]);
 var options, errors = 0;
 
-var g = new Getopt(['deps']);
-var optcur;
-
-loop:
-while (optcur = g.optind, g.getopt(process.argv)) {
-  switch (g.opt) {
-    case 'deps':
```
Comment (arv) [inline build/expand-js-template.js]: What was the intent here?

**Comment by arv** [comment]
LGTM

Make sure you try this with make clean too.

File: build/expand-js-template.js (hunk 1 above)
Comment (johnjbarton) [inline build/expand-js-template.js]: I have no idea.

This code is now only used for 
```
./src/outputgeneration/regexpuRewritePattern.js-template.js
./src/outputgeneration/SourceMapIntegration.js-template.js
```

**Comment by johnjbarton** [comment]
Yes, I thought the Travis failure was unrelated since it looked so bizarre.

**Comment by arv** [comment]
Still LGTM

When you bizarre Travis failure you can restart the build and see if it happens again.

**Pull request closed without merging**